components/menu-dropdown/button-trigger.jsx

```diff
@@ -126,12 +126,11 @@ class Trigger extends React.Component {
 			onMouseDown,
 			onMouseEnter,
 			onMouseLeave,
-			openOn,
 			triggerRef,
 			triggerClassName,
 			...deprecatedPropsFromMenuDropdown
 		} = this.props;
-
+		let { openOn } = this.props;
```

Comment:
minor - Might be better to create a `const` var directly: `const openOn = this.props.openOn === 'hover' ? 'click' : this.props.openOn` instead of doing this in 2 steps.

Reply:
Yes. That would be compact and more readable. Thanks.